.github/workflows/build.yml

```diff
@@ -0,0 +1,52 @@
+name: Build
+
+on: push
+
+jobs:
+    build:
+        name: Build
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                package: [".", "v2"]
+        defaults:
+            run:
+              working-directory: ${{ matrix.package }}
+        steps:
+            -   name: Set up Go 1.15
+                uses: actions/setup-go@v2
+                with:
+                    go-version: ^1.15
+
+            -   name: Check out
+                uses: actions/checkout@v2
+
+            -   name: Get dependencies
+                run: go get -v ./...
+
+            -   name: Enforce Go Formatted Code
+                run: "! go fmt ./... 2>&1 | read"
+
+            -   name: Identify Vet Errors
+                run: go vet ./...
+
+            -   name: Build
+                run: go build ./...
+
+            -   name: Test
+                run: go test -race -v ./...
+
+            -   name: CI fail
+                if: ${{ failure() }}
+                run: |
+                    curl --request POST '${{ secrets.WEBHOOK_URL }}' \
```

Comment:
Where are we sending these alerts?

Reply:
The alerts can be sent to any communication platform (like Slack, Microsoft Teams, etc.,) as long as matches the API contract. For now, it can be sent to slack and teams based on the `WEBHOOK_URL` configured.
@maknahar

Reply:
I do not think a common payload will work for every platform. Did you try sending it to Slack and Team? Can you add a sample of how they look in the PR?

Reply:
@ashwinspg In case you missed it.

Reply:
>I do not think a common payload will work for every platform. Did you try sending it to Slack and Team? Can you add a sample of how they look in the PR?

I have added the [description and drawback](https://github.com/RealImage/moviebuff-sdk-go/pull/14#issue-522098612) of using our current implementation. Please look into it.

@maknahar @svetha-cvl @suryapandian @sasipriyags